pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -191,6 +191,12 @@ func (pas *PodAutoscalerStatus) CanMarkInactive(idlePeriod time.Duration) bool {
 	return pas.inStatusFor(corev1.ConditionTrue, idlePeriod)
 }
 
+// IsActivatingTimeout checks whether the pod autoscaler has been in activating
+// for at least the specified timeout period.
+func (pas *PodAutoscalerStatus) IsActivatingTimeout(activatingTimeout time.Duration) bool {
```

Comment:
Probably here and in the config-map, etc "Activation Timeout" is a more correct name.